data/chapters/australia-sydney.json

```
@@ -12,7 +12,7 @@
       "website": "https://rladiessydney.org/"
     },
     "organizers": {
-      "current": ["Jenny Richmond"],
+      "current": ["Giorgia Mori"],
       "former": []
```

```suggestion
      "former": ["Jenny Richmond"]
```

We always retain the names of former organisers, both to show appreciation but also to have someone to contact in case we need something from them in the hand-over :)

---

Got it thanks!